lib/location/decorate.mjs

```diff
@@ -147,7 +147,5 @@ async function trash() {
       id: this.id
     }))
 
-  this.layer.reload()
```

Comment:
Don't we require this reload when we delete a location from a mvt layer? Such as a catchment statistics. Otherwise the geometry will still remain on the canvas without reloading the layer?

Reply:
Yes it is. The reload is in the location remove method which is called from the location trash(delete) method.

Reply:
So it does delete it from the dataset but it still remains on the canvas!

Reply:
https://github.com/GEOLYTIX/xyz/assets/43169158/32223440-b625-4576-b479-bdfcab165c88

Reply:
This should be resolved in latest commit. MVT layer were not reloaded.

Reply:
@dbauszus-glx Shap!